npc/events/aprilfools_2010.txt

```diff
@@ -0,0 +1,788 @@
+//===== rAthena Script =======================================
+//= April Fools Day Event (2010)
+//===== By: ==================================================
+//= Kisuka
+//===== Current Version: =====================================
+//= 1.0
+//===== Compatible With: =====================================
+//= rAthena Project
+//===== Description: =========================================
+//= [Official Conversion]
+//= Help Romeo lie to everyone.
+//===== Additional Comments: =================================
+//= 1.0 First version. [Kisuka]
+//============================================================
```

Comment:
```
//===== rAthena Script =======================================
//= Overall Script Title
//===== Description: =========================================
//= Description here.
//===== Changelogs: ==========================================
//= 1.0 Blah [Author]
//============================================================
```
Do you mind adjusting your script headers to this format? I know not all of our scripts follow it yet but it's the new header format we use now.

Reply:
Adjusted them to include 'changelogs' instead. Let me know if they still need more tweaking.

Reply:
Currently none that I'm aware of. Pretty much all new scripts that have been written use the new header. No one's taken the time to go back yet.

Reply:
fixed.